substrate/frame/support/src/lib.rs

```diff
@@ -2138,14 +2152,29 @@ pub mod pallet_macros {
 	/// ```
 	/// #[frame_support::pallet(dev_mode)]
 	/// mod pallet {
+	/// # 	use frame_support::pallet_prelude::*;
+	/// # 	use frame_system::pallet_prelude::*;
+	/// #
```

Comment:
macro is hard enough, we don't need to hide lines.
(I see it is the case elsewhere, but I still think it is not good to hide such lines)

```suggestion
	///   	use frame_support::pallet_prelude::*;
	///   	use frame_system::pallet_prelude::*;
	/// 
```